dlib/CMakeLists.txt

```diff
@@ -784,6 +784,8 @@ if (NOT TARGET dlib)
 
    endif ()  ##### end of if NOT DLIB_ISO_CPP_ONLY ##########################################################
 
+   # OpenCV4 is cleaning up the old C API compat.
+   target_compile_definitions(dlib PRIVATE -DCV__DISABLE_C_API_CTORS)
```

Comment:
There are no .cpp files in dlib that #include any opencv code.  So it seems like this cmake statement should have no effect.  Or is there something I'm overlooking?

Reply:
Then in this case it's a noop.  Please remove it from the PR :)